packages/trip-form/i18n/en-US.yml

```diff
@@ -52,3 +52,29 @@ otpUi:
     use: Use # as in "Use bus, train, subway".
     useCompanies: Use companies # as in :"Use companies: Spin, Lime, Bolt"
     walkOnly: Walk Only
+  ModeSelector:
+    labels:
+      TRANSIT: Transit
+      BICYCLE: Bike
+      WALK: Walk
+      CAR: Drive
+      RENT: Rent
+    settings:
+      walkTolerance-labelLow: Less walking
+      walkTolerance-labelHigh: More walking
+      walkTolerance-label: Walk Tolerance
+      bikeTolerance-labelLow: Less biking
+      bikeTolerance-labelHigh: More biking
+      bikeTolerance-label: Bike Tolerance
+      allowBikeRental-label: Bike Rental
+      carTolerance-label: Drive Tolerance
+      carTolerance-labelHigh: Less Driving
+      carTolerance-labelLow: More Driving
+      wheelchair-label: Accessible Routing
+      addBus-label: Bus
+      addRail-label: Train
+      addSubway-label: Subway
+      addTram-label: Tram
+      # Following ones used for storybook
+      busColor-label: Bus Color
+      busColor-options-blue: Blue
```

Comment:
I would advise against shipping these dev-only items. A better approach would be to default to the `label` attribute that was defined in the storybook mode config if a string could be found.